tests/compilation_tests/mag_heading_test.py

```diff
@@ -4,16 +4,16 @@
 import numpy as np
 from ahrs.filters import EKF
 import csv
-import os 
+import os
```

Comment:
adding a comment here because i cant figure out how to put it above:

running this test _only works_ when i remove the wmm2020 import; wmm2020 isn't even used in this test so i think that we can just remove it

otherwise, the pathing worked on my computer!

Reply:
good point! lets do this in a separate pr, could you make it @juliehelenek ?